playbooks/ansible-network-asa-appliance/files/bootstrap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# NOTE(pabelanger): This needs to move into proper zuul-job, however until we
+# can use vyos_config on zuul-executor, we need to run as nested.
+- hosts: appliance
+  gather_facts: false
+  vars:
+    ansible_password: admin
+    ansible_user: admin123
+    auth_pass: cisco
+  tasks:
+    - name: lookup SSH public key
+      set_fact:
+        _ssh_key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
+
+    - name: Setup new user
+      become: true
+      become_method: enable
+      asa_config:
+        lines:
+          - username zuul privilege 15 nopassword
+          - "username zuul sshkey {{ _ssh_key.split()[0] }} {{ _ssh_key.split()[1] }}"
```

Comment:
This also seems wrong as per Cisco ASA doc plz ref: [Configure SSH Access](https://www.cisco.com/c/en/us/td/docs/security/asa/asa912/configuration/general/asa-912-general-config/admin-management.html)